components/navbar/NavbarExploreOptions.vue

```diff
@@ -27,29 +27,33 @@
       <span
         v-for="option in filteredChains"
         :key="option.value"
-        class="menu-item mr-2"
+        class="menu-item mr-2 is-capitalized"
         :value="option.value"
         @click="setSelectedChain(option.value)">
-        {{ option.text }}
+        {{ getChainNameByPrefix(option.value) }}
       </span>
     </div>
   </div>
 </template>
 
 <script lang="ts" setup>
+import { getChainNameByPrefix } from '@/utils/chain'
+
 const { urlPrefix, setUrlPrefix } = usePrefix()
 const { availableChains } = useChain()
 
 const emit = defineEmits(['closeMobileNavbar'])
 
 const filteredChains = computed(() => {
-  return availableChains?.value.filter((chain) => {
-    return ['ksm', 'bsx', 'rmrk'].includes(chain.value as string)
-  })
+  return ['ahk', 'ahp', 'ksm']
```

Comment:
what this array define?

Reply:
It defines the options that we would like to show on the menu and also the order of them.